src/TMxpMudlet.h

```diff
@@ -85,6 +87,8 @@ class TMxpMudlet : public TMxpClient
     void setBold(bool bold) override { isBold = bold; }
     void setItalic(bool italic) override { isItalic = italic; }
     void setUnderline(bool underline) override { isUnderline = underline; }
+    void setStyle(const QString& val) override {mxpStyle = val; }
+    const QString &getStyle() override {return mxpStyle;}
```

Comment:
Just space them out a bit so they have the same spacing as everything else:
```suggestion
    void setStyle(const QString& val) override { mxpStyle = val; }
    const QString &getStyle() override { return mxpStyle; }
```